package.json

```diff
@@ -916,49 +926,71 @@
     "@typescript-eslint/parser": "^2.12.0",
     "chai": "^4.2.0",
     "codecov": "^3.5.0",
+    "css-loader": "^3.5.3",
     "cytoscape": "^3.14.0",
     "cytoscape-dagre": "^2.2.2",
     "decache": "^4.5.1",
     "eslint": "^6.7.2",
     "eslint-plugin-header": "^3.0.0",
+    "file-loader": "^6.0.0",
     "glob": "^7.1.3",
     "istanbul": "^0.4.5",
     "mocha": "^6.2.2",
     "mocha-jenkins-reporter": "^0.4.3",
+    "node-sass": "^4.14.1",
+    "npm-run-all": "^4.1.5",
     "proxyquire": "^2.1.0",
     "remap-istanbul": "^0.13.0",
     "rimraf": "^3.0.2",
+    "sass-loader": "^8.0.2",
     "simple-svg-tools": "^1.1.12",
     "sinon": "^7.3.2",
     "sinon-chai": "^3.3.0",
     "source-map-support": "^0.5.16",
+    "style-loader": "^1.2.1",
+    "thread-loader": "^2.1.3",
     "tmp": "0.1.0",
     "ts-loader": "^6.2.1",
     "typescript": "^3.5.2",
     "vscode-test": "^1.3.0",
     "walker": "^1.0.7",
-    "webpack": "^4.41.6",
+    "webpack": "^4.43.0",
     "webpack-cli": "^3.3.11"
   },
   "dependencies": {
     "@kubernetes/client-node": "^0.10.2",
+    "@patternfly/patternfly": "2.71.5",
```

Comment:
All dependencies that we use in any webview should be moved to `devDependencies` as we compiled them it webpack bundle, and we do not need to distribute them inside `vsix` package